frontend/src/layout/lemonade/SideBar/index.scss

```diff
@@ -0,0 +1,53 @@
+@import '~/vars';
```

Comment:
I think these files (`frontend/src/layout/lemonade/SideBar/index.tsx` & `frontend/src/layout/lemonade/SideBar/index.scss`) are a mistake, we already have them in master as `frontend/src/layout/lemonade/SideBar/SideBar.tsx`), and those seem to work correctly. Didn't remove them myself in case you made a change that should be included.